test_polygon.py::test_ocean_tx__create fails with 'execution reverted'

#### Bug report

I had a PR where all tests passed, [here](https://github.com/oceanprotocol/ocean.py/actions/runs/3768602564). 

Merging into trunk, the tests failed, [here](https://github.com/oceanprotocol/ocean.py/actions/runs/3768794247). [Here's the log](https://github.com/oceanprotocol/ocean.py/actions/runs/3768794247/jobs/6407488863).

So it's sporadic failures.

### Traceback

```text
____________________________ test_ocean_tx__create _____________________________

tmp_path = PosixPath('/tmp/pytest-of-runner/pytest-0/test_ocean_tx__create1')
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7ff4af66b6[70](https://github.com/oceanprotocol/ocean.py/actions/runs/3768794247/jobs/6407488863#step:7:71)>

    @pytest.mark.integration
    def test_ocean_tx__create(tmp_path, monkeypatch):
        """On Polygon, do a simple Ocean tx: create"""
        monkeypatch.delenv("ADDRESS_FILE")
        # setup
        connect_to_network("polygon-main")
        util.set_aggressive_gas_fees()
    
        config = get_config_dict("polygon-main")
        ocean = Ocean(config)
    
        accounts.clear()
        (alice_wallet, _) = util.get_wallets()
    
        # Do a simple-as-possible test that uses ocean stack, while accounting for gotchas
>       util.do_ocean_tx_and_handle_gotchas(ocean, alice_wallet)

tests/integration/remote/test_polygon.py:30: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/enforce_typing/decorator.py:29: in wrapper
    return func(*args, **kwargs)
tests/integration/remote/util.py:116: in do_ocean_tx_and_handle_gotchas
    raise (e)
tests/integration/remote/util.py:108: in do_ocean_tx_and_handle_gotchas
    data_nft = ocean.data_nft_factory.create(
ocean_lib/models/data_nft_factory.py:34: in create
    return data_nft_args.deploy_contract(self.config_dict, wallet)
ocean_lib/models/arguments.py:56: in deploy_contract
    receipt = data_nft_factory.deployERC[72](https://github.com/oceanprotocol/ocean.py/actions/runs/3768794247/jobs/6407488863#step:7:73)1Contract(
/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/brownie/network/contract.py:1864: in __call__
    return self.transact(*args)
/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/brownie/network/contract.py:1[73](https://github.com/oceanprotocol/ocean.py/actions/runs/3768794247/jobs/6407488863#step:7:74)7: in transact
    return tx["from"].transfer(
/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/brownie/network/account.py:644: in transfer
    receipt, exc = self._make_transaction(
/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/brownie/network/account.py:752: in _make_transaction
    exc = VirtualMachineError(e)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = VirtualMachineError(ValueError("Execution reverted during call: 'execution reverted'. This transaction will likely revert. If you wish to broadcast, include `allow_revert:True` as a transaction parameter."))
exc = "Execution reverted during call: 'execution reverted'. This transaction will likely revert. If you wish to broadcast, include `allow_revert:True` as a transaction parameter."

    def __init__(self, exc: ValueError) -> None:
        self.txid: str = ""
        self.source: str = ""
        self.revert_type: str = ""
        self.pc: Optional[int] = None
        self.revert_msg: Optional[str] = None
        self.dev_revert_msg: Optional[str] = None
    
        try:
            exc = exc.args[0]
        except Exception:
            pass
    
        if not (isinstance(exc, dict) and "message" in exc):
>           raise ValueError(str(exc)) from None
E           ValueError: Execution reverted during call: 'execution reverted'. This transaction will likely revert. If you wish to broadcast, include `allow_revert:True` as a transaction parameter.

/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/brownie/exceptions.py:93: ValueError
----------------------------- Captured stdout call -----------------------------
alice_wallet.address = '0x8bE6c8126C4383[74](https://github.com/oceanprotocol/ocean.py/actions/runs/3768794247/jobs/6407488863#step:7:75)5c25F3b3Ca[76](https://github.com/oceanprotocol/ocean.py/actions/runs/3768794247/jobs/6407488863#step:7:77)D73E3Caf8544'
bob_wallet.address = '0xD6764a9F3387B14692CC783BcF5604a23EBb[77](https://github.com/oceanprotocol/ocean.py/actions/runs/3768794247/jobs/6407488863#step:7:78)9f'
Call create() from data NFT, and wait for it to complete...
...

=========================== short test summary info ============================
FAILED tests/integration/remote/test_polygon.py::test_ocean_tx__create - Valu...
```

--- a/tests/integration/remote/util.py
+++ b/tests/integration/remote/util.py
@@ -127,6 +127,7 @@ def error_is_skippable(error_s: str) -> bool:
         or "No contract deployed at" in error_s
         or "nonce too low" in error_s
         or "Internal error" in error_s
+        or "execution reverted" in error_s
         or "No data was returned - the call likely reverted" in error_s
     )
 

